controller/PublicController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace EmilieSchott\BlogPHP\Controller;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\OAuth;
+use League\OAuth2\Client\Provider\Google;
+
+class PublicController {
+    public function homePage(object $twig, string $action) {
+        echo $twig->render('homeView.html.twig', ['action' => $action]);
+    }
+    
+    public function contactMe() {
+        if (
+            empty($_POST['name']) ||
+            empty($_POST['firstName']) ||
+            empty($_POST['email']) ||
+            empty($_POST['message']) ||
+            !filter_var($_POST['email'], FILTER_VALIDATE_EMAIL)
+        ) {
+            echo "No arguments Provided!";
+            return false;
+        }
+            
+        $name = htmlspecialchars($_POST['name']);
+        $firstName = htmlspecialchars($_POST['firstName']);
+        $email_address = htmlspecialchars($_POST['email']);
+        $message = htmlspecialchars($_POST['message']);
+
+        date_default_timezone_set('Etc/UTC');
```

Comment:
This should be set in your index.php, for all the website

Reply:
Ok, I will remember it and do the modification.